packages/react-components/src/search/EventSearch/views/Table/index.js

```diff
@@ -142,7 +142,7 @@ const defaultTableConfig = {
       trKey: 'filters.measurementOrFactTypes.name',
       value: {
         key: 'measurementOrFactTypes',
-        formatter: (value, item) => <>{value.join(', ')}</>
+        formatter: (value, item) => <>{value || [].join(', ')}</>
```

Comment:
As in @qifeng-bai 's branch this looks wrong to me.
Isn't this the same as `value || ''` and hence nonsense

Do you not mean something like`(value || []).join(', ')` ? Or is it me who are not aware of the order of operators.

Reply:
you are right. I pulled this from @qifeng-bai 's branch. Updated now to the above expression.